discovery/mock_test.go

```diff
@@ -56,12 +56,22 @@ func (p *mockPeer) Address() net.Addr { return nil }
 func (p *mockPeer) QuitSignal() <-chan struct{} {
 	return p.quit
 }
+func (p *mockPeer) GenDeliveryScript() ([]byte, error) {
```

Comment:
use the "embed interface" trick to avoid all these empty methods?  😄

Reply:
Yeah will do! Just wanted to check in lnd-dev before I started using embedded interfaces.